### PR TITLE
[Use cases] Add a use case for normal access pattern.

### PR DESCRIPTION
**Pull request by dajiaji**
Thanks @tomoyukilabs for drafting the CG report.

As I mentioned in [my proposal](#https://github.com/dajiaji/proposals/blob/abstract_proposal/draft_proposal_supporting_local_https_communication.md), I think we can categorize two device access patterns principle for local HTTPS as follows. 

> - Normal access pattern: the device has web contents and a user types the address of the device (e.g., ‘https://device.local’) on the browser directly and receives the contents.
> - Cross-origin access pattern: the device has API endpoints and a web frontend loaded on a browser from the internet (hereafter, simply called ‘web service’) accesses the APIs with a browser API (e.g., fetch API) and receives the contents.

Although [The current CG report draft](#https://httpslocal.github.io/usecases/) includes many use cases for the cross-origin access pattern,
there is no use case for the normal access pattern.
So I'd like to add a use case for the normal access pattern by modifying UC-07.
It is because the UC-07 treats home gateways which can be accessed by way of the normal access pattern and, in addition, I don't want to increase the number of use cases any more.

Could you check the PR?

### CODE REVIEW

**Comment by daniel-kun** [comment]
@dajiaji Good find! Apparently I didn't read the use case good enough, because I thought it would explicitly address the local, direct HTTPS access via browser. But apparently it didn't, so thanks for this clarification.
This is the most important use-case for the numerous Home Automation gateways that offer a Web Frontend for configuration and use.

File: index.bs [hunk 1]
```diff
@@ -111,12 +111,26 @@ she downloads the photos and posts them to the NAS device.
...
 ## UC-07: Secure offline communication for home automation ## {#uc-07}
 
-A user sets up a home automation gateway is under normal circumstances using
-HTTPS to securely accept commands via a remote server. In some cases the
-gateways internet connection is interrupted but local communication between a
-wall mounted control surface, app, or similar should stil be given. This becomes
-very important when we consider that devices that can be controlled this way
-include door locks and security cameras.
+A user wants to set up a new home device (e.g., a Wi-Fi access point/router,
+home automation gateway) in his/her home using the device web interface.
```
Comment (tomoyukilabs) [inline index.bs]: s/using the device web interface/via its web interface/

File: index.bs [hunk 2]
```diff
@@ -111,12 +111,26 @@ she downloads the photos and posts them to the NAS device.
...
+User typically connects the device with a laptop or a smartphone and then
+launches a given URL (e.g., https://home-router.local, or https://192.168.1.1)
+on the laptop or smart phone’s web browser. Since the new device does not have
```
Comment (tomoyukilabs) [inline index.bs]: My suggestion is: "User typically opens a given URL for the device (e.g., https://home-router.local, or https://192.168.1.1) in a laptop or smart phone’s web browser."

File: index.bs [hunk 3]
```diff
@@ -111,12 +111,26 @@ she downloads the photos and posts them to the NAS device.
...
+a valid web PKI certificate, the web browser displays a warning that typically
+says, “The connection is not secure” but provides an option to the user to
```
Comment (tomoyukilabs) [inline index.bs]: s/that typically says,/like/

File: index.bs [hunk 4]
```diff
@@ -111,12 +111,26 @@ she downloads the photos and posts them to the NAS device.
...
+approve the access or add the URL as an exception to the access list.
```
Comment (tomoyukilabs) [inline index.bs]: s/to the user to approve/to ask the user to approve/
s/access list/whitelist/

File: index.bs [hunk 5]
```diff
@@ -111,12 +111,26 @@ she downloads the photos and posts them to the NAS device.
...
+Once user takes the action, the web browser treats the device as a trusted one
+and does not display the warning in subsequent attempts. Additionally, if the
+home device supports functionality such as, SSO (Single Sign On) the user can
```
Comment (tomoyukilabs) [inline index.bs]: s/Single Sign On/Single Sign-On/

File: index.bs [hunk 6]
```diff
@@ -111,12 +111,26 @@ she downloads the photos and posts them to the NAS device.
...
+use an identity (ID) provider's account (e.g., Google, Facebook, and so on)
+to sign in the device’s web interface and provide the needed trust.
+
+Similar secure access is required when a home gateway, which is typically
+controlling other home devices (e.g., door locks, security cameras), uses HTTPS
+for securely accepting commands via a remote server gets temporarily disconnected
```
Comment (tomoyukilabs) [inline index.bs]: s/a remote server gets/a remote server. However, when a home getaway happens to get/

File: index.bs [hunk 7]
```diff
@@ -111,12 +111,26 @@ she downloads the photos and posts them to the NAS device.
...
+from the internet. In such scenarios, a local HTTPS communication from user’s
```
Comment (tomoyukilabs) [inline index.bs]: s/from the Internet./from the Internet, those home devices become inaccessible./

File: index.bs [hunk 8]
```diff
@@ -111,12 +111,26 @@ she downloads the photos and posts them to the NAS device.
...
+local device (e.g., smart phone or a control device) is essential to control
+and operate the home device.
```
Comment (tomoyukilabs) [inline index.bs]: s/device./devices directly through HTTPS connections over the local network./

**Comment by dajiaji** [comment]
Hmm, I couldn't notice your mentions and reviews till now...

**Comment by dajiaji** [comment]
@daniel-kun Thanks for your comments! @tomoyukilabs Thanks. I revised UC-07 according to your comments.

**Comment by tomoyukilabs** [comment]
@dajiaji LGTM. I've merged the PR. I'll generate index.html with Bikeshed later. Thanks!

**Pull request merged**